scalafix-tests/src/test/resources/checkSyntax/RemoveXmlLiterals.source

```diff
@@ -0,0 +1,112 @@
+rewrites = [RemoveXmlLiterals]
+
+<<< don't add import when not needed
+object A {}
+>>>
+object A {}
+
+<<< single line, no splice
+object A {
+  <div></div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  xml"<div></div>"
```

Comment:
Nice! 😍  I think users will appreciate having single quotes for simple xml literals.